### PR TITLE
Use `timeseriesPanel` instead of `panel` when creating panels

### LINKED ISSUE #44: Cortex-Mixins generates Angular Graph panels which will be deprecated in Grafana 11
As per the title, the cortex-mixins currently generate the Angular graph panel types which will be deprecated in Grafana 11 later this year. They should be replaced by the React based Timeseries panels.

### PR DESCRIPTION
**Pull request by CharlieTLe**
**What this PR does**:
Update panel to timeseries panel

**Which issue(s) this PR fixes**:
Fixes: https://github.com/cortexproject/cortex-jsonnet/issues/44
Depends on: https://github.com/cortexproject/cortex-jsonnet/pull/57

**Checklist**
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: cortex-mixin/dashboards/writes.libsonnet [hunk 1]
```diff
@@ -34,7 +34,7 @@ local utils = import 'mixin-utils/utils.libsonnet';
          showTitle: false,
        })
       .addPanel(
-        $.panel('Samples / sec') +
+        $.timeseriesPanel('Samples / sec') +
```
Comment (friedrichg) [inline cortex-mixin/dashboards/writes.libsonnet:37]: I tested this with
https://gist.github.com/friedrichg/116c9c75a81f4044800dbbaec44bff30
and doesn't look correct yet. 

And this is what I see
Screenshot 2024-10-11 at 18 59 25

and the old one looks like this on the same grafana with old dashboards

Screenshot 2024-10-11 at 18 53 21

**Comment by CharlieTLe** [comment]
@friedrichg I think I fixed it, could you take another look? 🙏🏼 
Screenshot 2024-10-17 at 7 33 31 PM

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,7 @@
 * [CHANGE] Target 3M memory series per ingester instead of 1.5M
 * [CHANGE] Update jsonnet-libs to Fri Jul 19 12:51:49 2024 #57
 * [CHANGE] Increase CortexProvisioningTooManyWrites alert threshold to 160e3
+* [CHANGE] Use `timeseriesPanel` instead of `panel` when creating panels #58
 * [ENHANCEMENT] Configure `-ingester.client.grpc-compression` to be `snappy-block`
 * [ENHANCEMENT] Support Grafana 11 in Cortex Service Scaling Dashboard
```
Comment (friedrichg) [inline CHANGELOG.md]:
```suggestion
* [ENHANCEMENT] Support Grafana 11 in all dashboards
```

**Review by friedrichg — APPROVED** [review]
Thank you! awesome work

**Pull request merged**